### ISSUE TITLE
[Magiclysm] Dreadful presence infinite loop

### ISSUE THREAD

**Issue by KorGgenT** [CleverRaven/Cataclysm-DDA#42787]
# Describe the bug
When stepping into a dragon's dreadful precense, it freezes you forever.

# Steps To Reproduce
Walk into a dreadful presence aura created by an adult black dragon

# Expected behavior
maybe some nausea and slight reduced speed

# Versions and configuration

 Exp. Build #10902
# Additional context
Didn't encounter this myself, making this bug report from watching Vormithrax's stream.

**Comment by Fris0uman** [comment]
Start game
Spawn Adult black dragon
Get one shot by breath attack
Repeat

I've tried this 7 times, including one time with debug invicibility and got hit by the dreadfull aura only once and it didn't trigger an infinite loop

**Comment by KorGgenT** [comment]
hmm, have you tried with the high acid resistance aura and armor without debug invincibility? because i think the debug invincibility will cause issues

**Comment by KorGgenT** [comment]
Duplicate of #40083
